packages/astro/src/vite-plugin-build-html/extract-assets.ts

```diff
@@ -54,6 +55,10 @@ function isAsset(node: Element) {
   if (!path) {
     return false;
   }
+  // Windows fix: if path starts with C:/, avoid URL parsing
+  if (windowsPathRE.test(path)) {
```

Comment:
This is the dumbest part: I couldn’t find a way around `C:/Users/…` URLs hitting here. Which, yes, `new URL('C:/Users/…')`, does generate a URL where the protocol is `c:` 🤦🏻

So I just did a RegEx test for letter + colon + slash at the start. I’m not sure of a better way to handle this but open to suggestions